### PR TITLE
Call formatter_create in #configure instead of #start in filter_stdout

### PR DESCRIPTION
**Pull request by cosmo0920**
See: https://github.com/fluent/fluentd/pull/1059#discussion_r76362903

### CODE REVIEW

File: lib/fluent/plugin/filter_stdout.rb [hunk 1]
```diff
@@ -34,10 +34,10 @@ class StdoutFilter < Filter
     def configure(conf)
       compat_parameters_convert(conf, :inject, :formatter)
       super
+      @formatter = formatter_create(conf: @config.elements('format').first, default_type: DEFAULT_FORMAT_TYPE)
     end
 
     def start
```
Comment (tagomoris) [inline lib/fluent/plugin/filter_stdout.rb]: It's possible to remove whole `#start` because it's just calling `super`.

Comment (cosmo0920) [inline lib/fluent/plugin/filter_stdout.rb]: I've removed this.

**Comment by tagomoris** [comment]
LGTM

**Comment by cosmo0920** [comment]
Thanks!

**Pull request merged**